packages/devtools_app/lib/src/config_specific/import_export/import_export.dart

```diff
@@ -117,3 +133,25 @@ abstract class ExportController {
     return jsonEncode(_contents..addAll({activeScreenId: contents}));
   }
 }
+
+class OfflineModeController {
+  bool get offlineMode => _offlineMode;
```

Comment:
should this be a ValueListenable?
Existing logic seemed like it was calling setState on changes to this.

Reply:
done